plugin/pkg/auth/authorizer/rbac/bootstrappolicy/testdata/cluster-roles.yaml

```diff
@@ -1051,6 +1051,13 @@ items:
     - create
     - patch
     - update
+  - apiGroups:
```

Comment:
Why are these changes in the same commit?

Reply:
Good call, I've split them into two separate commits now, thanks!